### ISSUE TITLE
Test inputs to low_rank routines

### ISSUE THREAD

**Issue by disa-mhembere** [xdata-skylark/libskylark#38]
I would like test [low_rank.hpp](https://github.com/xdata-skylark/libskylark/blob/lowrank/ml/low_rank.hpp) for correctness using [low_rank.cpp](https://github.com/xdata-skylark/libskylark/blob/lowrank/examples/low_rank.cpp). @kenclarkson could you provide me with reasonable values for parameters `k`, `epsilon`, `m`, `ml`, `mr` given some square matrix dimension `n`.  This is from your low ranks psd manuscript. Unsure `low_rank_sym` and `low_rank_PSD` function correctly.

**Comment by kenclarkson** [comment]
I didn't understand: low_rank_hpp is an implementation of the low-rank PSD approximation algorithm in that manuscript? (Probably you said already)

**Comment by kenclarkson** [comment]
(low_rank.hpp)

**Comment by disa-mhembere** [comment]
Yes that's correct -- the low-rank PSD approx you wrote with David. Sorry if that was unclear.

Now I want to verify correctness so I would some example parameters I can test with.
